interface/ceed-tensor.c

```diff
@@ -23,26 +23,25 @@
   @brief Create a CeedTensorContract object for a CeedBasis
 
   @param[in]  ceed     Ceed object where the CeedTensorContract will be created
-  @param[in]  basis    CeedBasis for which the tensor contraction will be used
   @param[out] contract Address of the variable where the newly created CeedTensorContract will be stored.
 
   @return An error code: 0 - success, otherwise - failure
 
   @ref Backend
 **/
-int CeedTensorContractCreate(Ceed ceed, CeedBasis basis, CeedTensorContract *contract) {
+int CeedTensorContractCreate(Ceed ceed, CeedTensorContract *contract) {
   if (!ceed->TensorContractCreate) {
     Ceed delegate;
 
     CeedCall(CeedGetObjectDelegate(ceed, &delegate, "TensorContract"));
     CeedCheck(delegate, ceed, CEED_ERROR_UNSUPPORTED, "Backend does not support TensorContractCreate");
-    CeedCall(CeedTensorContractCreate(delegate, basis, contract));
+    CeedCall(CeedTensorContractCreate(delegate, contract));
     return CEED_ERROR_SUCCESS;
   }
 
   CeedCall(CeedCalloc(1, contract));
   CeedCall(CeedReferenceCopy(ceed, &(*contract)->ceed));
-  CeedCall(ceed->TensorContractCreate(basis, *contract));
```

Comment:
Removing this argument probably belongs as a separate PR

Reply:
Yeah sorry, this was needed in order to implement the approach in the first commit, but that ended up being reverted for the approach using `CeedBasisGetTensorContract`. I'll pull it out.

Reply:
Fixed, see https://github.com/CEED/libCEED/pull/1360